### ISSUE TITLE
[Tarea 1] Duda testcase 02.ManiobrasSinEfecto

### ISSUE THREAD

**Issue by tomasalvarez13** [IIC2113-2022-2/Syllabus#11]
Hola, tengo una duda con respecto al siguiente cas de prueba:
![image](https://user-images.githubusercontent.com/48598610/193644605-8640bb85-32b4-45c4-9641-719cda3f9c84.png)
Al final de la foto el jugador UNDERTAKER juega la carta Russian Leg Sweep, la cual tiene fortitude 2, pero mi oponente tiene fortitude 0, porque le acabo de hacer daño con Wrist Lock (entonces al jugar esa carta yo soy el que suma 2 a su fortitude). Entonces no entiendo si yo me estoy confundiendo, y el requisito para poder jugar una carta es que MI fortitude debe ser mayor o igual al de la carta qu euiero usar, o si el test está mal.

**Comment by RodrigoToroIcarte** [comment]
Sí, claro. El requisito para jugar una carta es que el fortitude del jugador sea mayor o igual al fortitude requerido por la carta que quiere jugar.

En el test case, UNDERTAKER tiene la carta "Russian Leg Sweep" pero inicialmente no la puede jugar porque UNDERTAKER tiene 0F y esa carta pide 2F. Así que UNDERTAKER juega un "Wrist Lock [0F/2D]". Luego de ejecutar el daño, UNDERTAKER tiene 2F y ahora sí puede jugar el 'Russian Leg Sweep" (y lo juega).

¿Esto responde tu pregunta? :)

**Comment by tomasalvarez13** [comment]
Perfecto muchas gracias
